interface/src/app/analyze/analyze.tpl.html

```diff
@@ -23,8 +23,8 @@ <h1 class="page-header">
             </div>
             <div class="media-body">
               <button type="button" class="btn"
-              ng-class="sb.has_item(search_item)?'disabled':''"
-              ng-click="sb.add_item(search_item)">
+              ng-class="sb.hasItem(search_item)?'disabled':''"
```

Comment:
:+1: on the change to camelCase. I was wondering if the line with ``sb.hasItem(search_item)?'disabled':''`` might be to much logic for the template and might be better to move into the controller though?